builtin/mainmenu/settings/dlg_settings.lua

```diff
@@ -46,49 +57,6 @@ local function add_page(page)
 end
 
 
-local change_keys = {
-	query_text = "Controls",
-	requires = {
-		keyboard_mouse = true,
-	},
-	get_formspec = function(self, avail_w)
-		local btn_w = math.min(avail_w, 3)
-		return ("button[0,0;%f,0.8;btn_change_keys;%s]"):format(btn_w, fgettext("Controls")), 0.8
-	end,
-	on_submit = function(self, fields)
-		if fields.btn_change_keys then
-			core.show_keys_menu()
-		end
-	end,
-}
-
-
-add_page({
-	id = "accessibility",
-	title = fgettext_ne("Accessibility"),
-	content = {
-		"language",
-		{ heading = fgettext_ne("General") },
-		"font_size",
-		"chat_font_size",
-		"gui_scaling",
-		"hud_scaling",
-		"show_nametag_backgrounds",
-		{ heading = fgettext_ne("Chat") },
-		"console_height",
-		"console_alpha",
-		"console_color",
-		{ heading = fgettext_ne("Controls") },
-		"autojump",
-		"safe_dig_and_place",
-		{ heading = fgettext_ne("Movement") },
-		"arm_inertia",
-		"view_bobbing_amount",
-		"fall_bobbing_amount",
-	},
-})
-
-
 local function load_settingtypes()
```

Comment:
https://github.com/minetest/minetest/blob/4c9be808a7222e82e04c24d927150664773ad5be/builtin/mainmenu/settings/settingtypes.lua#L443-L444

The mod list is no longer loaded twice on startup, but it's still loaded twice. Why not use `pkgmgr.global_mods` here?

```lua
pkgmgr.load_all()
local mods = pkgmgr.global_mods:get_list()
```

Reply:
I don't know why this uses get_mods rather than `global_mods`, but I've changed it now